contracts/WhitelistedCommitment.sol

```diff
@@ -133,9 +133,11 @@ contract WhitelistedCommitment is PublicCommitment {
     }
 
     function WhitelistedCommitment(uint256 _startDate, uint256 _endDate, uint256 _minCommitment, uint256 _maxCommitment,
-        uint256 _minTicket, uint256 _ethEurFraction, ITokenWithDeposit _ethToken, LockedAccount _lockedAccount, Curve _curve)
+        uint256 _minTicket, uint256 _ethEurFraction, ITokenWithDeposit _ethToken, LockedAccount _lockedAccount,
+        Curve _curve, IAccessPolicy _policy)
```

Comment:
Convention elsewhere is to put forwarded constructor arguments in the order of the base classes, in  this case `_policy` would come first.